oauth2client/client.py

```diff
@@ -858,6 +897,10 @@ def _do_revoke(self, http_request, token):
     query_params = {'token': token}
     token_revoke_uri = _update_query_params(self.revoke_uri, query_params)
     resp, content = http_request(token_revoke_uri)
+
+    if six.PY3 and isinstance(content, bytes):
```

Comment:
I'm going to put this in another PR. FWIW this is also pointed out to me that we detect and convert to/from bytes in a very inconsistent way.